packages/otel/src/main.ts

```diff
@@ -0,0 +1,12 @@
+import { type SugaredTracer } from '@opentelemetry/api/experimental';
+import { GET_TRACER, SHUTDOWN_TRACERS } from './constants.js';
+
+export const getTracer = async (name?: string, version?: string): Promise<SugaredTracer | undefined> => {
+    // @ts-ignore 
```

Comment:
To avoid using `@ts-ignore`, in other places I did this: https://github.com/netlify/primitives/blob/b996796a147ba57f31ff2d7f1a9e19ce5fba24e1/packages/cache/src/fetchwithcache.ts#L143

Reply:
Nice, updated to use similar approach